src/tests/functional/plugin/shared/include/behavior/ov_plugin/core_integration.hpp

```diff
@@ -385,6 +386,18 @@ TEST_P(OVClassSetDevicePriorityConfigTest, SetConfigAndCheckGetConfigNoThrow) {
     ASSERT_EQ(devicePriority, configuration[ov::device::priorities.name()].as<std::string>());
 }
 
+TEST_P(OVClassSetTBBForceTerminatePropertyTest, SetConfigNoThrow) {
+    ov::Core ie = createCoreWithTemplate();
+
+    bool value = true;
+    OV_ASSERT_NO_THROW(ie.set_property(deviceName, ov::force_tbb_terminate(false)));
```

Comment:
I suppose this property is NOT related to specific device and must be used as `ie.set_property( ov::force_tbb_terminate(false));`

In this case we don't need changes in `SetConfigForPlugins`

Reply:
Yes, It has been resolved in https://github.com/openvinotoolkit/openvino/pull/12022